sortfilterproxymodel.qbs

```diff
@@ -31,6 +31,7 @@ Product
     }
 
     cpp.includePaths: [ "." ]
+    cpp.cppFlags: "-std=c++11"
```

Comment:
@crozeg Please retry this with `cpp.cxxLanguageVersion: "c++11"`. I know you mentioned that you tried that, but it should definitely work... See https://doc.qt.io/qbs/cpp-module.html - this would be a better and more portable solution.